docs/packaging/advanced-config/local-repository.md

```diff
@@ -45,8 +45,15 @@ Every time you run `go-task build-local`, all `eopkg` files in the local reposit
 
 There are some important things to know when working with local repositories, as they may lead to issues later on.
 
-- `solbuild` will use your version of a package from the local repository regardless of whether there's a higher release in the Solus repository. Therefore you should only use `go-task build-local` when required and also remove old packages from the local repository when they are no longer needed.
-- If the package is already installed in the `solbuild` image, the release must be higher for it to be installed.
+- `solbuild` will use your version of a package from the local repository regardless of whether there's a higher release in the Solus repository. Therefore:
+  -  Only use `go-task build-local` when required
+  -  Remove old packages from the local repository when they are no longer needed, and rebuild its index with the following commands
+  ```bash
+  cd /var/lib/solbuild/local
+  sudo rm *.eopkg
+  sudo eopkg index --skip-signing /var/lib/solbuild/local/ --output /var/lib/solbuild/local/eopkg-index.xml
```

Comment:
I don't this is needed. `build-local` automatically re-indexes when used anyway. 
Maybe it would be needed if you only removed *some* local packages, and then wanted to use the local repository for installation.

Reply:
Also, in the spirit of `go-task` commands I just learned: `go-task build-localindex`

Reply:
Thanks for the feedback. I think it's safer to recommend re-indexing whenever someone removes eopkg files. It will help prevent someone's repo being in an unknown state if they come to us for help.
Good point about the go-task, I've updated the command to reflect that.

Reply:
I've also changed a couple of other manual commands to be go-task commands, thanks to ermo's reminder